docs/configuration.md

```diff
@@ -55,6 +55,31 @@ Then those packages are usable directly:
    plt.show();
 ```
 
+### pip packages
+
+⚠ This feature is experimental. You won't have the same user-experience as when using conda/mamba in a "normal" setup ⚠
+
+`xeus-python` provides a way to install packages with pip.
+
+There are a couple of limitations that you should be aware of:
```

Comment:
Would it also support installing via a requirements file?

```yaml
- pip:
  - -r requirements.txt
```

Since this is also a common use case. If it's not supported we could add it to the list of limitations.